OpenOversight/app/main/views.py

```diff
@@ -565,6 +565,85 @@ def submit_department_images(department_id=1):
     return render_template('submit_department.html', department=department)
 
 
+def check_input(str_input):
+    if str_input is None or str_input == "Not Sure":
+        return ""
+    else:
+        return str(str_input).replace(",", " ")  # no commas allowed
+
+
+@main.route('/download/department/<int:department_id>', methods=['GET'])
+@limiter.limit('5/minute')
+def download_dept_csv(department_id):
+    department = Department.query.filter_by(id=department_id).first()
+    records = Officer.query.filter_by(department_id=department_id).all()
+    if not department or not records:
+        abort(404)
+    dept_name = records[0].department.name.replace(" ", "_")
+    first_row = "id, last, first, middle, suffix, gender, "\
+                "race, born, employment_date, assignments, notes\n"
+
+    assign_dict = {}
+    assign_records = Assignment.query.all()
+    for r in assign_records:
+        if r.officer_id not in assign_dict:
+            assign_dict[r.officer_id] = []
+        assign_dict[r.officer_id].append("(#%s %s %s %s %s)" % (check_input(r.star_no), check_input(r.rank), check_input(r.unit), check_input(r.star_date), check_input(r.resign_date)))
+
+    notes_dict = {}
```

Comment:
Let's remove the notes from the CSV, the reason being is that the notes are actually for admins/area coordinators to add private notes to a profile, so we don't want to expose them to regular users